### DISCUSSION TITLE
How do I use Filter() in supabase-csharp?

### DISCUSSION THREAD

**Discussion by dimunyo** [supabase/supabase#10598] [Questions] [answered]
![image](https://user-images.githubusercontent.com/55717823/204438451-844d6c7d-a9b0-497b-a27e-cba2fdf558ca.png)
Hi, I`m trying to get the username with the filter function, I don't know the right syntax.

**Comment by GaryAustin1** [comment, marked as answer]
I don't use and don't know if it matters but the example in the repository shows: `Postgrest.Constants.Operator.Equals,` not `Constants.Operator.Equals`.
